pkg/solana/transmissions_cache.go

```diff
@@ -19,19 +19,26 @@ type TransmissionsCache struct {
 	*client.Cache[Answer]
 }
 
-func NewTransmissionsCache(transmissionsID solana.PublicKey, chainID string, cfg config.Config, reader client.Reader, lggr logger.Logger) *TransmissionsCache {
+func NewTransmissionsCache(transmissionsID solana.PublicKey, chainID string, cfg config.Config, getReader GetReader, lggr logger.Logger) *TransmissionsCache {
```

Comment:
why are we passing `getReader` to just wrap it into `getAccountReader`? it would be cleaner to just pass account reader